tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html

```diff
@@ -55,7 +55,7 @@
         {% include_block page.body %}
 
         {% if page.related_blog_posts %}
-            {% include "patterns/atoms/motif-heading/motif-heading.html" with heading="More" heading_level=2 classes="section-title--related-posts motif-heading--two" aria_label="Related blog posts" %}
+            {% include "patterns/atoms/section-title/section-title.html" with title="More" aria_label="Related blog posts" classes="section-title--related-posts" %}
```

Comment:
I think this is the same place I also updated the title so we'll need to rebase.

Reply:
Now rebased.